x.crypto.chacha20: enables supports for 64-bit counter (#24053)
vlang/v: vlib/x/crypto/chacha20/README.md

## chacha20

Chacha20 (and XChacha20) stream cipher encryption algorithm in pure V.
Its mostly based on [RFC 8439](https://datatracker.ietf.org/doc/html/rfc8439) 
and inspired by Go version of the same library.

## Status

> This module already support for cipher with 32-bit counter, and recently expanded 

Reviewer: ```suggestion
This module already supports a 32-bit counter mode, and recently expanded
```

Author: Done on last commit

> to support for 64-bit counter. 

Reviewer: ```suggestion
to support a 64-bit counter mode. 
```

Author: done

> This features already mainlined at the time of writing:

Reviewer: ```suggestion
The features already mainlined at the time of writing (2025/03/27):
```

Author: Changed
